src/FSAL/FSAL_CORTXFS/handle.c

```diff
@@ -1870,12 +1870,12 @@ static fsal_status_t kvsfs_handle_digest(const struct fsal_obj_handle *obj_hdl,
 	assert(fh_desc);
 	assert(obj_hdl);
 
-	if (output_type != FSAL_DIGEST_NFSV4) {
+	/*if (output_type != FSAL_DIGEST_NFSV4) {
```

Comment:
Instead of commenting, just remove it. Keep a comment stating that we accept both kind of digests